tests/spectron/resizeWindows.spectron.js

```diff
@@ -0,0 +1,61 @@
+const Application = require('./spectronSetup');
+const robot = require('robotjs');
+
+let app = new Application({});
+
+describe('Tests for Resizing windows', () => {
+
+    let originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = Application.getTimeOut();
+
+    beforeAll((done) => {
+        return app.startApplication().then((startedApp) => {
+            app = startedApp;
+            done();
+        }).catch((err) => {
+            done.fail(new Error(`Unable to start application error: ${err}`));
+        });
+    });
+
+    afterAll((done) => {
+        if (app && app.isRunning()) {
+            // resize to default size
+            app.browserWindow.getBounds().then((bounds) => {
+                let x = bounds.x - (defaultWidth - bounds.width);
+                let y = bounds.y - (defaultHeight - bounds.height);
+                robot.moveMouse(bounds.x, bounds.y);
+                robot.mouseToggle("down");
+                robot.dragMouse(x, y);
+                robot.mouseToggle("up");
+            })
+            //close app
+            jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout;
+            app.stop().then(() => {
+                done();
+            }).catch((err) => {
+                done();
+            });
+        }
+    });
+
+    it('should be minimized up to 300px', (done) => {
```

Comment:
can we add a comment as to why we are running this test?

Reply:
Added comment.